datasette/utils/__init__.py

```diff
@@ -307,11 +308,18 @@ def make_dockerfile(
 COPY . /app
 WORKDIR /app
 {spatialite_extras}
+{environment_variables}
 RUN pip install -U {install_from}
 RUN datasette inspect {files} --inspect-file inspect-data.json
 ENV PORT 8001
 EXPOSE 8001
 CMD {cmd}""".format(
+        environment_variables="\n".join(
+            [
+                "ENV {} {}".format(key, value)
```

Comment:
Need to consider escaping here. At the very least the value should be in single quotes.

Reply:
https://twitter.com/simonw/status/1147620863841144832

Reply:
The bash manual says "A single quote may not occur between single quotes, even when preceded by a backslash" - I'm going to flat out ban single quotes here. http://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html